### ISSUE TITLE
Vehicle Speed Broken

### ISSUE THREAD

**Issue by Creeper258YT** [SmartlyDressedGames/Unturned-3.x-Community#1319]
A lot of vehicles seems affected by a speed problem, where their speed get stuck at 24kph, a lot are modded but some vanilla vehicles are affected too. It happened to a lot of people, some people also tried to do fixes for some big car mods. Here a list, so you can test vanilla and modded vehicles that weren't working for me.

Modded Vehicle : 
- Doge Charter by IMIVEOOO_ [To workshop](https://steamcommunity.com/sharedfiles/filedetails/?id=1658442827)

Vanilla Vehicle : 
- Firetruck
- Police Car (only when car takes damage)

Here are the logs files (have fun reading them they are each about 2500 lines)
[Client.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/3970780/Client.log)
[Client_Prev.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/3970781/Client_Prev.log)

**Comment by SDGNelson** [comment]
In short: tonight I'm uploading a tiny adjustment of default car wheel mass from 1 to 2.

For custom vehicles: I strongly recommend checking out this page which allows much more extensive customization of the car physics: https://github.com/SmartlyDressedGames/U3-Docs/blob/master/VehiclePhysicsProfile.md

Thanks for the report. I should have picked up on this sooner, but I got the impression that past discussions were from prior to the patch that reverted the default car physics profile. Behavior of cars pre-19.19.0 and post-19.19.1 should be identical, as the `DefaultProfile_Car.asset` was emptied of overrides. I reviewed all the 19.19 commits and am confident the behavior is identical aside from a change to how brakes work, and also added some editor logging to double-check that was the case. These vehicles still seem to be experiencing the wheel collider changes from one of the 2018.4 Unity updates, but raising the default wheel mass override from 1 to 2 seems to make them more stable.
